handler/src/main/java/io/netty/handler/ssl/SslContextSupplier.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import io.netty.util.Supplier;
+import io.netty.util.internal.ObjectUtil;
+
+/**
+ * A {@link Supplier} for {@link SslContext}s.
+ */
+public final class SslContextSupplier implements Supplier<SslContext> {
```

Comment:
`SslContextSupplier` -> `FixedSslContextSupplier`.

Can you make this more general purpose?

```java
public final class FIxedSupplier<T> implements Supplier<T> {
   private final T object;
...
}
```